directory.ts

```diff
@@ -22,11 +22,28 @@ export class Directory {
     });
   }
 
-  async getUserByUserID(subject: string): Promise<User> {
-    const user = await this.client.object({
-      type: "user",
-      key: subject,
-    });
+  async getUserByIdentity(identity: string): Promise<User> {
+    const relation = await this.client.relation(
+      {
+        subject: {
+          type: 'user',
+          key: identity
```

Comment:
The key should be left empty. After my changes it will no longer contain the PID (i.e. `identity`).